Modelica/Electrical/Machines/Examples/InductionMachines/IMC_YDarc.mo

```diff
@@ -0,0 +1,144 @@
+  Modelica.Blocks.Sources.BooleanStep booleanStepYD[m](each startTime=
+        tStart2) annotation (Placement(transformation(extent={{-80,-40},{-60,
+            -20}})));
+  Modelica.Mechanics.Rotational.Components.Inertia loadInertia(J=JLoad)
+    annotation (Placement(transformation(extent={{40,-60},{60,-40}})));
+  Modelica.Mechanics.Rotational.Sources.QuadraticSpeedDependentTorque
+    quadraticLoadTorque(
+    w_nominal=wLoad,
+    TorqueDirection=false,
+    tau_nominal=-TLoad,
+    useSupport=false) annotation (Placement(transformation(extent={{90,-60},{70,
+            -40}})));
+  parameter Utilities.ParameterRecords.IM_SquirrelCageData aimcData "Induction machine data"
+    annotation (Placement(transformation(extent={{-20,-90},{0,-70}})));
+  Polyphase.Basic.Resistor resistor(m=m, R=fill(RLine, m)) annotation (
+      Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={0,80})));
+  Polyphase.Basic.Inductor inductor(m=m, L=fill(LLine, m)) annotation (
+      Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={0,50})));
+initial equation
+  inductor.i[1:2] = zeros(2);
+  aimc.is = zeros(3);
+  aimc.ir = zeros(2);
+equation
+  connect(star.pin_n, ground.p)
+    annotation (Line(points={{-70,90},{-80,90}}, color={0,0,255}));
+  connect(sineVoltage.plug_n, star.plug_p)
+    annotation (Line(points={{-40,90},{-50,90}},      color={0,0,255}));
+  connect(loadInertia.flange_b, quadraticLoadTorque.flange)
+    annotation (Line(points={{60,-50},{70,-50}}));
+  connect(booleanStep.y, idealCloser.control) annotation (Line(points={{-59,30},
+          {-20,30},{-20,20},{-12,20}},    color={255,0,255}));
+  connect(booleanStepYD.y, switchYDwithArc.control)
+    annotation (Line(points={{-59,-30},{-22,-30}}, color={255,0,255}));
+  connect(idealCloser.plug_n, currentQuasiRMSSensor.plug_p)
+    annotation (Line(points={{0,10},{0,0}},         color={0,0,255}));
+  connect(switchYDwithArc.plug_sn, aimc.plug_sn)
+    annotation (Line(points={{-16,-40},{-16,-40}}, color={0,0,255}));
+  connect(switchYDwithArc.plug_sp, aimc.plug_sp)
+    annotation (Line(points={{-4,-40},{-4,-40}}, color={0,0,255}));
+  connect(switchYDwithArc.plugSupply, currentQuasiRMSSensor.plug_n)
+    annotation (Line(points={{-10,-20},{0,-20}}, color={0,0,255}));
+  connect(aimc.flange, loadInertia.flange_a) annotation (Line(
+      points={{0,-50},{40,-50}}));
+  connect(sineVoltage.plug_p, resistor.plug_p)
+    annotation (Line(points={{-20,90},{0,90}}, color={0,0,255}));
+  connect(resistor.plug_n, inductor.plug_p)
+    annotation (Line(points={{0,70},{0,60}}, color={0,0,255}));
+  connect(inductor.plug_n, idealCloser.plug_p)
+    annotation (Line(points={{0,40},{0,30}}, color={0,0,255}));
+  annotation (experiment(
+      StopTime=2.5,
+      Interval=0.0001,
+      Tolerance=1e-06,
+      __Dymola_Algorithm="Dassl"),                                      Documentation(
```

Comment:
This should be removed.

Reply:
What do you mean by "This"?

Reply:
Please delete `__Dymola_Algorithm="Dassl"`.